### PR TITLE
ServiceCatalog: update owners

### PR DESCRIPTION
**Pull request by jboyd01**

### CODE REVIEW

File: examples/service-catalog/OWNERS [hunk 1]
```diff
@@ -1,6 +1,8 @@
 reviewers:
   - bparees
   - pmorie
+  - jboyd01
 approvers:
   - bparees
   - pmorie
+  - jboyd01
```
Comment (bparees) [inline examples/service-catalog/OWNERS]: i think it's up to @jwmatthews (or maybe @pmorie ?) whether you should have approval or not.  review seems fine.

**Comment by jboyd01** [comment]
added jpeeler under reviewers

**Comment by jboyd01** [comment]
the service-catalog failing job is a known issue, I'm investigating.  It shouldn't hold up this PR.

**Pull request merged**